scripts/checkpoint_converter.py

```diff
@@ -22,7 +22,7 @@
 
 Example usage:
 
-python3 checkpoint_converter /path/to/checkpoint.ckpt /path/to/output \
+$ python checkpoint_converter /path/to/checkpoint.ckpt /path/to/output \
```

Comment:
why is this `python` but the instructions in the downloader say `python3 checkpoint_downloader /checkpoint/url/ /path/to/output`? I know 0 python

Reply:
oversight